entities/build.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+set -e
+
+# Absolute path to the directory containing this script
+DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
```

Comment:
Actually suggest leaving this to a follow-up to minimize changes made together with the code move.